doc/user/content/sql/create-sink.md

```diff
@@ -36,8 +36,17 @@ Field | Use
 ------|-----
 **KAFKA BROKER** _host_ | The Kafka broker's host name.
 **TOPIC** _topic&lowbar;prefix_ | The prefix used to generate the Kafka topic name to create and write to.
+**WITH OPTIONS (** _option&lowbar;_ **)** | Options affecting sink creation. For more details see [`WITH` options](#with-options).
 **CONFLUENT SCHEMA REGISTRY** _url_ | The URL of the Confluent schema registry to get schema information from.
 
+### `WITH` options
+
+The following options are valid within the `WITH` clause.
+
+Field | Value type | Description
+------|------------|------------
+`replication_factor` | `int` | The replication factor for the sink Kafka topic. This defaults to 1 if not set.
```

Comment:
this feels a little tautological but idk if being more verbose is better?

Reply:
I think this is pretty good! We could maybe add a little more context (e.g. provide a verb) to make it feel less tautological, e.g.

> Set the sink Kafka topic's replication factor. Defaults to 1.

This is nice because it gives the option an action that the user can leverage--not that "set" is the correct one, there might well be something clearer.

Reply:
thats super helpful thanks. made the change as suggested